packages/components/psammead-brand/src/__snapshots__/index.test.jsx.snap

```diff
@@ -14,138 +14,100 @@ exports[`Brand should render correctly 1`] = `
 
 .c0 {
   background-color: #B80000;
-  height: 5rem;
+  height: 3.5rem;
   width: 100%;
+  padding: 0 0.5rem;
 }
 
-.c1 {
-  max-width: 80rem;
-  margin: 0 auto;
-}
-
-.c3 {
-  padding-top: 0.5rem;
+.c2 {
   display: inline-block;
 }
 
-.c4 {
-  display: block;
-  padding-bottom: 1.5rem;
+.c3 {
+  box-sizing: content-box;
+  fill: #FFFFFF;
+  max-width: 17.5rem;
+  min-width: 11.25rem;
+  padding-top: 1rem;
+  padding-bottom: 0.75rem;
+  width: 100%;
 }
 
-.c2:hover .c4,
-.c2:focus .c4 {
+.c1:hover .c3,
+.c1:focus .c3 {
   -webkit-text-decoration: none;
   text-decoration: none;
   border-bottom: 0.25rem solid #FFFFFF;
 }
 
-.c5 {
-  display: block;
-  height: 1.5rem;
-  width: 10.50rem;
-  margin-top: 1.25rem;
-  fill: #fff;
-}
-
-@media (max-width:25rem) {
-  .c0 {
-    padding: 0 0.5rem;
-  }
-}
-
-@media (min-width:25rem) {
+@media (min-width:37.5rem) {
   .c0 {
+    height: 5rem;
     padding: 0 1rem;
   }
 }
 
-@media (min-width:62.9375rem) {
-  .c0 {
-    padding: 0 1rem;
+@media (min-width:37.5rem) {
+  .c3 {
+    padding-top: 1.75rem;
+    padding-bottom: 1.5rem;
   }
 }
 
 @media screen and (-ms-high-contrast:active),print {
-  .c5 {
+  .c3 {
     fill: windowText;
   }
 }
 
 <div
   className="c0"
 >
-  <div
-    className="c1"
+  <a
+    className="c1 c2"
+    href="https://www.bbc.co.uk/news"
   >
-    <a
-      className="c2 c3"
-      href="https://www.bbc.co.uk/news"
+    <svg
+      aria-hidden="true"
+      className="c3"
+      focusable="false"
+      height={24}
+      viewBox="0 0 167.95 24"
+      xmlns="http://www.w3.org/2000/svg"
     >
-      <span
-        className="c4"
+      <g
+        fillrule="evenodd"
       >
-        <svg
-          aria-hidden="true"
-          className="c5"
-          focusable="false"
-          viewBox="0 0 167.95 24"
-          xmlns="http://www.w3.org/2000/svg"
-        >
-          <g
-            fillrule="evenodd"
-          >
-            <path
-              d="M84.32"
-            />
-          </g>
-        </svg>
-        <span
-          className="c6"
-        >
-          Default Brand Name
-        </span>
-      </span>
-    </a>
-  </div>
+        <path
+          d="M84.32"
+        />
+      </g>
+    </svg>
+    <span
+      className="c4"
+    >
+      Default Brand Name
+    </span>
+  </a>
 </div>
 `;
 
 exports[`Brand should render correctly with svg not provided 1`] = `
 .c0 {
   background-color: #B80000;
-  height: 5rem;
+  height: NaNrem;
```

Comment:
Is this an expected value? Is there a reason we don't pass a default value through?

Reply:
This is expected as the test scenario is purely to check that the component doesn't throw a runtime error when no SVG is provided. That said the CSS could be handled slightly better, I'll take a second look ...